commit 01fbb4b82126134ca995f90b2b42057a3af9a123
Author: Sven Roederer <freifunk@it-solutions.geroedel.de>
Date:   Sun Apr 24 22:38:26 2016 +0200

    freifunk: fix lookup of community-name
    
    restorethe lookup of the freifunk community-name stored in
    uci "freifunk.community.name".
    In https://github.com/openwrt/luci/commit/9780ee382e72f8a5fb69e337a3fcc51fc0914883
    the value canged to the complete path of the community-profile, e.g.
    "/etc/config/profile_berlin". This causes lookup problems on other
    pages, like "mod-freifunk -> overview -> index". And as the option suggests
    it's the community-name not the community-profile path.

diff --git a/feeds/luci/modules/luci-mod-freifunk/luasrc/model/cbi/freifunk/basics.lua b/feeds/luci/modules/luci-mod-freifunk/luasrc/model/cbi/freifunk/basics.lua
index 0d3d971..b08366d 100644
--- a/feeds/luci/modules/luci-mod-freifunk/luasrc/model/cbi/freifunk/basics.lua
+++ b/feeds/luci/modules/luci-mod-freifunk/luasrc/model/cbi/freifunk/basics.lua
@@ -16,7 +16,7 @@ community.rmempty = false
 local profile
 for profile in fs.glob(profiles) do
 	local name = uci:get_first(profile, "community", "name") or "?"
-	community:value(profile, name)
+	community:value(string.gsub(profile, "/etc/config/profile_", ""), name)
 end
 
 
